src/libs/actions/Report.js

```diff
@@ -1580,6 +1580,63 @@ Onyx.connect({
     },
 });
 
+/**
+ * Creates an optimistic IOU reportAction
+ *
+ * @param {String} type IOUReportAction type. Can be oneOf(create, decline, cancel, pay).
```

Comment:
**type of IOUReportAction**. Can be oneOf(create, decline, cancel, pay).

Reply:
or maybe just `IOUReportAction type`

Reply:
I'll keep it that way.